tests/README.md

```diff
@@ -51,14 +51,19 @@ The cluster created by the automation scripts are shared across all of the test
 ### Prerequisite
 Please read repo's [README.md](../README.md)
 #### Users and Tokens
-To execute the test cases, we need to prepare the [offline token](https://console.redhat.com/openshift/token/show). Get the offline token and export as an ENV variable **rhcs_TF_TOKEN**.
-* export rhcs_TF_TOKEN=<offline token>
+To execute the test cases, we need to prepare the [offline token](https://console.redhat.com/openshift/token/show). Get the offline token and export as an ENV variable 
 
+**rhcs_TF_TOKEN**.
+* export RHCS_TOKEN=<offline token>
+* export SDQE_ROSA_API=<offline token> (temporarily)
```

Comment:
Why do we need additional token?

Reply:
for this:
`versionsList := HELPER.GetGreaterOrEqualVersions(RosaConnection, majorVersion, profile.ChannelGroup, true, true)`

Reply:
the `RHCS_TOKEN` is a string, while this `SDQE_ROSA_API` is a connection obj

Reply:
oh, you are right
I fixed that :)